The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data/pgdata ... ok
creating subdirectories ... ok
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting dynamic shared memory implementation ... posix
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok

WARNING: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
syncing data to disk ... ok

Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data/pgdata -l logfile start

waiting for server to start....2025-02-14 09:00:11.376 UTC [47] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 09:00:11.385 UTC [48] LOG:  database system was shut down at 2025-02-14 09:00:11 UTC
2025-02-14 09:00:11.388 UTC [47] LOG:  database system is ready to accept connections
 done
server started
CREATE DATABASE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

waiting for server to shut down....2025-02-14 09:00:11.741 UTC [47] LOG:  received fast shutdown request
2025-02-14 09:00:11.742 UTC [47] LOG:  aborting any active transactions
2025-02-14 09:00:11.743 UTC [47] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
2025-02-14 09:00:11.743 UTC [49] LOG:  shutting down
2025-02-14 09:00:11.750 UTC [47] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2025-02-14 09:00:11.849 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-02-14 09:00:11.849 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2025-02-14 09:00:11.853 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 09:00:11.860 UTC [65] LOG:  database system was shut down at 2025-02-14 09:00:11 UTC
2025-02-14 09:00:11.864 UTC [1] LOG:  database system is ready to accept connections
2025-02-14 09:00:18.407 UTC [80] ERROR:  relation "_prisma_migrations" does not exist at character 126
2025-02-14 09:00:18.407 UTC [80] STATEMENT:  SELECT "id", "checksum", "finished_at", "migration_name", "logs", "rolled_back_at", "started_at", "applied_steps_count" FROM "_prisma_migrations" ORDER BY "started_at" ASC
2025-02-14 09:00:18.745 UTC [80] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 09:00:18.745 UTC [80] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_6e69bdc6-bf09-4b26-b9e2-e4621fcd9fd1" WITH (FORCE)
2025-02-14 09:00:19.290 UTC [80] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 09:00:19.290 UTC [80] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_ec3b90f1-3838-4442-b873-d2ef6e926241" WITH (FORCE)
2025-02-14 09:00:19.753 UTC [80] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 09:00:19.753 UTC [80] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_1452ce32-c0cd-43ca-8570-1a196a0618b0" WITH (FORCE)
2025-02-14 09:00:20.328 UTC [80] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 09:00:20.328 UTC [80] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_65f981fd-c587-4788-af5b-e4c2bd7edbf9" WITH (FORCE)
